testnets/terpnettestnet/assetlist.json

```diff
@@ -0,0 +1,50 @@
+{
+    "$schema": "../assetlist.schema.json",
+    "chain_name": "TerpNet",
+    "assets": [
+      {
+        "description": "Terp Token (TERP) is the Terp Network's native utility token, used as the primary means to govern, secure the blockchain, incentivize participants, and provide a default mechanism to store and exchange value.",
+        "denom_units": [
+          {
+            "denom": "uterpx",
+            "exponent": 0
+          },
+          {
+            "denom": "terp",
+            "exponent": 6
+          }
+        ],
+        "base": "uterpx",
+        "name": "Terp Testnet Token",
+        "display": "terp",
+        "symbol": "TERP",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terpnet/images/terp.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terpnet/images/terp.svg"
+        },
+        "coingecko_id": "terp-net-token"
+      },
+      {
+        "description": "Persy Gas Token (Persy) is the Terp Network's native utility token, used as the primary means to govern, secure the blockchain, incentivize participants, and provide a default mechanism to store and exchange value.",
+        "denom_units": [
+          {
+            "denom": "upersyx",
+            "exponent": 0
+          },
+          {
+            "denom": "persy",
+            "exponent": 6
+          }
+        ],
+        "base": "upersyx",
+        "name": "Persy Testnet Gas Token",
+        "display": "persy",
+        "symbol": "PERSY",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terpnet/images/persy.png",
```

Comment:
Can you upload these files, too, please, so they exist?
(i.e., terp.png, terp.svg, persy.png, persy.svg)

Reply:
got it, thank you.